src/simpleImg.js

```diff
@@ -38,27 +38,36 @@ export default class SimpleImg extends React.PureComponent<Props, State> {
   }
 
   componentDidMount() {
-    const cachedImagesRefString = window.sessionStorage.getItem('__REACT_SIMPLE_IMG__');
```

Comment:
we don't need to access `sessionStorage` in `componentDidMount` unless the config option to disable animation of cached images is set to tru